Makefile

```diff
@@ -9,7 +9,7 @@
 #
 # CGM_BASE_DIR = /path/to/installed/cgm
 
-include ${CGM_BASE_DIR}/lib/iGeom-Defs.inc
+-include ${CGM_BASE_DIR}/lib/iGeom-Defs.inc
```

Comment:
Not familiar with this syntax... does it just fail gracefully in the absence of the variable?